resources/authgear/templates/__default__/web/__password_policy.html

```diff
@@ -1,4 +1,5 @@
 {{ define "__password_policy.html" }}
+{{ $isNew := .IsNew }}
```

Comment:
This declaration is unnecessary. If we want to reference the fields of the root data object, we can use `{{ $.IsNew }}`.

Reply:
Removed the declaration